pkg/asset/installconfig/platformpermscheck.go

```diff
@@ -48,10 +48,18 @@ func (a *PlatformPermsCheck) Generate(dependencies asset.Parents) error {
 	platform := ic.Config.Platform.Name()
 	switch platform {
 	case aws.Name:
-		permissionGroups := []awsconfig.PermissionGroup{awsconfig.PermissionCreateBase, awsconfig.PermissionDeleteBase}
+		permissionGroups := []awsconfig.PermissionGroup{awsconfig.PermissionCreateBase}
+
+		// Delete is supported in all cases except C2S.
+		if ic.Config.AWS.Region != "us-iso-east-1" {
+			permissionGroups = append(permissionGroups, awsconfig.PermissionDeleteBase)
+		}
+
 		// If subnets are not provided in install-config.yaml, include network permissions
 		if len(ic.Config.AWS.Subnets) == 0 {
 			permissionGroups = append(permissionGroups, awsconfig.PermissionCreateNetworking, awsconfig.PermissionDeleteNetworking)
+		} else if ic.Config.AWS.Region != "us-iso-east-1" {
+			permissionGroups = append(permissionGroups, awsconfig.PermissionDeleteSharedNetworking)
```

Comment:
With this logic, the PermissionsDeleteNetworking would be added when installing to C2S when there are no subnets provided in the install config.
```suggestion
		usingExistingVPC := len(ic.Config.AWS.Subnets) != 0
		if !usingExistingVPC {
			permissionGroups = append(permissionGroups, awsconfig.PermissionCreateNetworking)
		}
		if !aws.C2SRegions.Has(ic.Config.AWS.Region) {
			if usingExistingVPC {
				permissionGroups = append(permissionGroups, awsconfig.PermissionDeleteSharedNetworking)
			} else {
				permissionGroups = append(permissionGroups, awsconfig.PermissionDeleteNetworking)
			}
		}
```

Reply:
> With this logic, the PermissionsDeleteNetworking would be added when installing to C2S when there are no subnets provided in the install config.

Oh, I see. I made the mistaken assumption all C2S was BYO networking.